pom.xml

```diff
@@ -12,6 +12,8 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+      <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Why are this changes required? Are they part of the requirements?

Reply:
Yeah! I've used maven, which allowed me to create a java project.  In order to run, I had to add those dependencies.